File: Unit1/java-programs/Programa 1.8.3/src/Fichero.java

import java.io.*;
public class Fichero{
	private static final String FICHERO="contador.txt";
	public static int leovalor(){
		FileReader fr;
		BufferedReader br;
		String linea=null;
		try{
		  fr=new FileReader(FICHERO);
		  br=new BufferedReader(fr);
		  linea=br.readLine();
		  fr.close();
		} catch (Exception e) {};
	 	return Integer.parseInt(linea);
	}
	public	static void escribovalor(int contador){
		FileWriter fw;
		PrintWriter pw;
		String linea=null;
		try{
		  fw=new FileWriter(FICHERO);
		  pw=new PrintWriter(fw);
		  pw.println(contador);
		  fw.close();
		} catch (Exception e) {};
	     
	}
}

File: Unit1/c-programs/Programa182/v3/fichero.c
#include <stdio.h>
#define FICHERO "contador.txt"

int leovalor(){
  FILE *fp;
  int contador;
  fp=fopen(FICHERO,"r");
  fscanf(fp,"%d",&contador);
  fclose(fp);
  return contador;
}
void escribovalor(int contador){
  FILE *fp;
  fp=fopen(FICHERO,"w");
  fprintf(fp,"%d",contador);
  fclose(fp);
}

